src/cocoa/toga_cocoa/libs/appkit.py

```diff
@@ -7,7 +7,7 @@
 
 from rubicon.objc import *
```

Comment:
At column 1: (F403) 'from rubicon.objc import *' used; unable to detect undefined names